lib/hyrax/collections_migration.rb

```diff
@@ -0,0 +1,14 @@
+module Hyrax
+  class CollectionsMigration
+    def self.run
+      ::Collection.all.each do |collection|
+        collection.members.each do |member|
+          member.member_of_collections << collection
+          member.save
```

Comment:
Of course if you throw exception or abort, your repo data is stuck in an unworkable state.  But it is better to make it obvious than silently propagate that error.